docs/contributing/installing.md

```diff
@@ -0,0 +1,52 @@
+---
+id: install
+title: Contributing
+sidebar_label: Install
+---
+
+# Install
```

Comment:
somehow this is both install and installing (per the filename) can you change the file name to install.md